### PR TITLE
[tests] Enable --blame for non-helix tests

### PR DESCRIPTION
**Pull request by radical**
Issue: https://github.com/dotnet/aspire/issues/3927

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/3931)

### CODE REVIEW

File: eng/.runsettings [hunk 1]
```diff
@@ -14,5 +14,22 @@
...
   </LoggerRunSettings>
-</RunSettings>
\ No newline at end of file
+  <DataCollectionRunSettings>
+    <DataCollectors>
+      <!-- Enables blame -->
+      <DataCollector friendlyName="blame" enabled="True">
+        <Configuration>
+          <!-- Enables crash dump, with dump type "Full" or "Mini".
+          Requires ProcDump in PATH for .NET Framework. -->
```
Comment (eerhardt) [inline eng/.runsettings:25]: This is kind of a weird comment. We don't do anything for .NET Framework. (I assume this means .NET Framework 4.x)

**Review by eerhardt — APPROVED** [review]
Code changes look fine if they work.

File: eng/.runsettings (hunk 1 above)
Comment (radical) [inline eng/.runsettings:25]:
```suggestion
          <!-- Enables crash dump, with dump type "Full" or "Mini". -->
```

File: eng/.runsettings [hunk 2]
```diff
@@ -14,5 +14,21 @@
...
+  <DataCollectionRunSettings>
+    <DataCollectors>
+      <!-- Enables blame -->
+      <DataCollector friendlyName="blame" enabled="True">
+        <Configuration>
+          <!-- Enables crash dump, with dump type "Full" or "Mini". -->
+          <CollectDump DumpType="Full" />
+          <!-- Enables hang dump or testhost and its child processes
+          when a test hangs for more than 10 minutes.
+          Dump type "Full", "Mini" or "None" (just kill the processes). -->
```
Comment (radical) [inline eng/.runsettings:28]:
```suggestion
          when a test hangs for more than 10 minutes. -->
```

**Pull request merged**